### ISSUE TITLE
doc error on events page

### ISSUE THREAD

**Issue by mvolkmann** [Polymer/polymer#3613]
In the doc page at https://www.polymer-project.org/1.0/docs/devguide/events.html, this line:

```
  el.addEventListener('click', function(){
```

should be

```
  el.addEventListener('click', function (event) {
```

**Comment by TimvdLippe** [comment]
Seems to be fixed at https://www.polymer-project.org/2.0/docs/devguide/events.html but the 1.0 version is still incorrect :cry:

**Comment by TimvdLippe** [comment]
Closing this as it is fixed in the 2.0 version. If you would still want to update the 1.0 version, please open a PR on https://github.com/polymer/docs
